src/test/java/web_pages/YandexHomePage.java

```diff
@@ -0,0 +1,68 @@
+package web_pages;
+
+import io.qameta.allure.Step;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import utils.Utils;
+
+import java.util.Properties;
+
+public class YandexHomePage {
```

Comment:
please revisit this page class, and apply all patterns, approaches, parametrization, code duplication, etc. that we've already studied.

Reply:
fixed